modules/sum.js

```diff
@@ -0,0 +1,21 @@
+import isArrayLike from './_isArrayLike.js';
+import values from './values.js';
+import cb from './_cb.js';
+import find from './find.js';
```

Comment:
```suggestion
import isArrayLike from 'underscore/modules/_isArrayLike.js';
import values from 'underscore/modules/values.js';
import 'underscore/modules/iteratee.js';
import _ from 'underscore/modules/underscore.js';
import find from 'underscore/modules/find.js';
```

Generally, Underscore-contrib modules shouldn't import internal modules from Underscore. In the case of `cb`, that's easy to avoid because it's equivalent to calling `_.iteratee` (except for an optimization). This is why I'm replacing the `_cb.js` import by `iteratee.js` and `underscore.js` imports above.

I couldn't quite make up my mind about what to do with `_isArrayLike.js`, so I just left it in place. Perhaps that module should become public.